knowledge-base/live-traffic-pane.md

```diff
@@ -57,19 +54,30 @@ Use the **>_ Terminal** button to capture traffic from a specific terminal insta
 
 [Learn more about the independent terminal capturing...]({%slug capture-terminal-traffic%})
 
+### Clearing Sessions
+
+The **Clear** button will remove all captured traffic from the **Live Traffic** grid. The action requires explicit confirmation. To remove only specific sessions, select the targeted sessions and use the **Clear** option from the context menu.
 
-### Removing All Sessions
+You can also clear only a specific set of sessions based on some pre-defined criteria. The complete list of **Clear xxx** options is available through the dropdown menu next to the **Clear** button, and the available options are as follows:
 
-The **Remove All** button deletes all captured traffic from the **Live Traffic** grid. The action requires explicit confirmation. To remove only specific sessions, select the desired ones and use the remove options from the context menu.
+- **Clear All**&mdash;Removes all entries from the traffic grid (an analog to the **Clear** button).
+- **Clear Closed**&mdash;Removes all sessions with session state "Closed".
```

Comment:
we can suggest them see sessions state by adding 'session state' column (its not added by default)